.travis.yml

```diff
@@ -1,10 +1,53 @@
 language: java
 
-jdk:
-  - oraclejdk8
-
 sudo: required
 
-install: true
-
-script: ./gradlew clean build -x test -x signArchives
+matrix:
+  include:
+    - language: java
+      install: true
+      os: osx
+      osx_image: xcode9.3
+      before_install:
+        - brew update
+        - brew outdated xctool || brew upgrade xctool
+        - npm install -g appium
+      script:
+        - ./gradlew clean -PexcludeTests=io.appium.java_client.ios.UIAutomationTest test --tests "io.appium.java_client.ios.*" --exclude-task signArchives
+    - language: java
+      install: true
+      os: osx
+      osx_image: xcode7.3
```

Comment:
Do we still need this?

Reply:
I kept it for testing legacy support. Have added a test in `UIAutiomationTest` class

Reply:
I'd rather try 8.3 then. Even if something fails there then I don't think one is gonna spend much time for invetigation

Reply:
Ok will make the necessary changes.